setup.cfg

```diff
@@ -2,7 +2,7 @@
 addopts = -v --junit-xml=test-report.xml
           --doctest-modules
           --cov=riotctrl --cov-branch
-          --cov-report=term --cov-report=xml --cov-report=html
+          --cov-report=term --cov-report=term-missing --cov-report=xml
```

Comment:
```suggestion
          --cov-report=term-missing --cov-report=xml
```

I don't think we need term if we use term-missing

Reply:
Done

Reply:
:scream: I helped!